### ISSUE TITLE
DBT Geometry and FBP Reconstruction

### ISSUE THREAD

**Issue by mozanunal** [odlgroup/odl#1597]
Hello, 
First of all, thank you for this great project.  We are a research team from Istanbul. We would like to use ODL project for **Digital Breast Tomosynthesis (DBT)** reconstruction. 

We have a look at the geometry documentation at https://odlgroup.github.io/odl/odl.tomo.geometry.html 
However, we could not figure out how we should arrange geometry for Digital Breast Tomosynthesis.
Do you have any example code or getting started guide for DBT reconstruction?

Thanks in advance.

**Comment by adler-j** [comment]
I must admit I'm a bit ignorant about DBT, do you have a more specific description of the geometry you want? My understanding is that you have a fixed detector and only move the source?

**Comment by mozanunal** [comment]
The devices which we have used so far are Hologic, Siemens, and GE. The detector collimators might vary from one device to another however the source moves in a linear directory for all devices as you mentioned.

**Comment by adler-j** [comment]
I don't think we have such geometries hardcoded into ODL right now since we haven't worked on it. To do it "nicely" you'd need to add a new `DivergentBeamGeometry`. `ConeBeamGeometry` is an example and while the [implementation](https://github.com/odlgroup/odl/blob/2a0c03c429ef6b44e6fbcd1c09456b8054016a09/odl/tomo/geometry/conebeam.py#L710) looks rather lengthy it's almost all documentation. If you want to implement one I can help out, otherwise you might need to wait a but until someone (me perhaps) does it.

**Comment by mozanunal** [comment]
I have another project on my focus now. I guess my project can take around 1 month. After that, I can focus on this one and try to implement it. Thanks for your kind interests and answers @adler-j @ozanoktem.

Another issue, I actually found an article about DBT reconstruction and I predicted that it is implemented with ODL toolbox (Since it is based on learned primal-dual). Thats why I want to contact you. I guess you are also in this publication if I am right? Maybe I should contact the first author.

https://arxiv.org/pdf/1808.04640.pdf
